packages/gatsby/src/internal-plugins/query-runner/file-parser.js

```diff
@@ -119,11 +118,13 @@ async function findGraphQLTags(file, text): Promise<Array<DefinitionNode>> {
           TaggedTemplateExpression(path) {
             if (
               (`descendant of query`,
+              // eslint-disable-next-line no-undef
```

Comment:
Couldn't figure out how to get babel-eslint to like the optional chaining here.

Reply:
Yeah, noticed that too — it must not be setup to use or repo babel setup

Reply:
There is one specific minor version that it works with...I'll find it I have it set in another project but don't remember off the top of my head :P

Reply:
that'd be ace, thanks :D

Reply:
its `8.2.1` and earlier i think, you can pin to that specific version tho, any higher and it's broken. https://github.com/babel/babel-eslint/issues/595

Reply:
Updated, thanks!